src/sparseml/transformers/utils/model.py

```diff
@@ -137,6 +147,10 @@ def question_answering_from_pretrained(
             model_name_or_path,
             **kwargs,
         )
+        if mask_name_or_path:
+            mask = AutoModelForQuestionAnswering.from_pretrained(mask_name_or_path)
```

Comment:
let's add an info log here, at minimum should say we are performing the mask transfer and give the model path, should also consider listing applied sparsities